src/libraries/sendtohelixhelp.proj

```diff
@@ -199,6 +199,8 @@
         <Targets Condition="'$(TargetArchitecture)' == 'x86'">x86</Targets>
         <AndroidPackageName>net.dot.%(Filename)</AndroidPackageName>
         <AndroidInstrumentationName>net.dot.MonoRunner</AndroidInstrumentationName>
+
```

Comment:
Nit: extra line

Reply:
yup that was intentional, to match the iOS/tvOS itemgroup.